src/main/java/at/aau/serg/websocketdemoserver/websocket/broker/WebSocketBrokerController.java

```diff
@@ -65,14 +66,13 @@ public String startGameForLobby(StartGameRequest startGameRequest) {
     }
 
     @MessageMapping("/play_card")
-    @SendTo("/topic/card_played")
     public String playCard(CardPlayRequest playCardRequest) throws Exception {
         Card card  = lobbyManager.cardPlayed(playCardRequest);
         CardPlayedRequest cardPlayedRequest = new CardPlayedRequest();
         cardPlayedRequest.setCardType(card.getCardType());
         cardPlayedRequest.setColor(card.getColor());
-        cardPlayedRequest.setValue(card.getValue());
-
+        cardPlayedRequest.setValue(String.valueOf(card.getValue()));
+        messagingTemplate.convertAndSend("/topic/card_played", cardPlayedRequest);
```

Comment:
In alter Version wurde auf card_played "objectMapper.writeValueAsString(cardPlayedRequest);"
Jetzt werden die selben Daten in der Mitte der Methode mittels convertAndSend geschickt. Warum / Wo ist der unterschied?

Reply:
Guter Punkt, es wird nur das "messagingTemplate..." benötigt, dieser Teil sendet auf alle Clients die subscribed sind, wir im nächsten Push so geändert das nur über diesen Call eine Antwort gesendet wird. :)